### PR TITLE
Add sudoer rights to default bookmarks

### PR DESCRIPTION
**Pull request by cmarianhodis**

### CODE REVIEW

File: Controller/AdminController.php [hunk 1]
```diff
@@ -24,6 +24,8 @@
 use BackBee\Bundle\AbstractAdminBundleController,
     BackBee\ClassContent\AbstractClassContent;
 
+use Symfony\Component\HttpFoundation\JsonResponse;
```
Comment (mickaelandrieu) [inline Controller/AdminController.php]: you dont use it, right ?

File: Controller/AdminController.php [hunk 2]
```diff
@@ -38,6 +40,26 @@ class AdminController extends AbstractAdminBundleController
...
+     * 
+     * @var bool
+     */
+    private $accessToDefault = false;
+
+    public function __construct(\BackBee\BBApplication $app)
+    {
+        parent::__construct($app);
+
+        $sudoers = $this->getApplication()->getConfig()->getSecurityConfig('sudoers') ? $this->getApplication()->getConfig()->getSecurityConfig('sudoers') : [];
```
Comment (mickaelandrieu) [inline Controller/AdminController.php]: Why not use a voter in this case ? Sounds terribly sad to sort of harcode security process here :( /c @eric-chau

Comment (crouillon) [inline Controller/AdminController.php]: You're right @mickaelandrieu , $this->isGranted('ROLE_SUDOER') should be enough

**Comment by crouillon** [comment]
@thx @cmarianhodis, :+1:

**Pull request merged**